src/core/utils/EventNormalizer.ts

```diff
@@ -902,10 +957,12 @@ export class EventNormalizer {
             (!this._compiledEvent || this._compiledEvent.key === 'a') &&
             this._isSelectAll(this._getRange())
         ) {
-            this._triggerEvent('selectAll', {
-                origin: this._compiledEvent ? 'keypress' : 'pointer',
-                target: this._selectAllOriginElement,
-            });
+            this._triggerEvent(
+                this._formatCustomEvent('selectAll', {
```

Comment:
miss (Though if you rename the functions like I suggested, you won't be able to miss it anymore since _formatCustomEvent would not exist anymore ;-))